app-emulation/euae/euae-0.8.29.recipe

```diff
@@ -0,0 +1,83 @@
+SUMMARY="E-UAE Experimental Ubiquitous Amiga Emulator"
+DESCRIPTION="This is a version of UAE, the Ubiquitous Amiga Emulator, with an \
+emulation core largely based on WinUAE. It attempts to bring many of the great\
+ features of WinUAE to non-Windows platforms. This version now finally has a n\
+ame, E-UAE, since that's what everybody was calling it anyway. The 'E' can sta\
+nd for anything you fancy. Experimental, extreme, exciting, egalitarian, eggpl\
+ant, . . .
+Currently it will build and run (with a varying degree of supported features) \
+on Linux and other Unices, Mac OS X, BeOS, AmigaOS itself (either for 68k mach\
+ines or PPC machines with AmigaOS 4.0 and the AmigaOS clones MorphOS and AROS)\
+. OS X requires either LibSDL or an X server for graphics output, but native g\
+raphics are supported on AmigaOS and BeOS, although, at the moment, SDL is als\
+o preferred on BeOS since the native driver is incomplete.
+E-UAE is open-source software and is made available under the terms of the GNU\
+ GPL. E-UAE is based on the work of dozens of contributors including Bernd Sch\
+ midt (the original author and maintainer of UAE), Bernie Meyer (the author of\
+  the x86 JIT compiler), Toni Wilen (the current maintainer of WinUAE), and ma\
+  ny more.
+To make full use of E-UAE you will need access to an image of some version of \
+the Amiga Kickstart ROM (although UAE does include a ROM emulation which may w\
+ork with some old games). The Amiga Kickstart ROMs are copyrighted material an\
+d may not be freely distributed (so don't ask me for a copy). If you don't hav\
+e an Amiga from which you can legally make a ROM image, a selection of ROM ima\
+ges are available for purchase online from Cloanto - publishers of the Amiga F\
+orever distribution."
```

Comment:
> Following [this](https://www.haiku-os.org/blog/humdinger/2016-05-28_contributing_pull_requests), I've (hopefully!) made those changes and amended my pull request.

All your changes are OK, with just one exception: the DESCRIPTION variable.
I'm sorry, I should have mentioned that we never split lines inside words. So if a word does not fit in the current line we end the line with one space and one backslash and move the word to the next line.
With this rule the first lines of the DESCRIPTION variable should look like:
```
DESCRIPTION="This is a version of UAE, the Ubiquitous Amiga Emulator, with an \
emulation core largely based on WinUAE. It attempts to bring many of the \
great features of WinUAE to non-Windows platforms. This version now finally \
has a name, E-UAE, since that's what everybody was calling it anyway. The 'E' \
can stand for anything you fancy. Experimental, extreme, exciting, \
egalitarian, eggplant, ...
```